docs-ref-services/key-vault.md

```diff
@@ -31,57 +30,51 @@ Use the management library to:
 ### Client library
 
 ```bash
-pip install azure-keyvault
+pip install  azure-keyvault-secrets azure-keyvault-keys azure-identity
 ```
 
 ## Examples
 
-The following examples use service principal authentication, which is the recommended sign in method
-for applications that connect to Azure. To learn about service principal authentication, see
-[Authenticate with the Azure SDK for Python](https://docs.microsoft.com/en-us/python/azure/python-sdk-azure-authenticate)
-
 Retrieve the public portion of an asymmetric key from a vault:
 
 ```python
-from azure.keyvault import KeyVaultClient
-from azure.common.credentials import ServicePrincipalCredentials
+from azure.identity import DefaultAzureCredential
+from azure.keyvault.keys import KeyClient
 
-credentials = ServicePrincipalCredentials(
-    client_id = '...',
-    secret = '...',
-    tenant = '...'
-)
+credential = DefaultAzureCredential()
 
-client = KeyVaultClient(credentials)
+key_client = KeyClient("https://<vaultname>.vault.azure.net", credential)
 
-# VAULT_URL must be in the format 'https://<vaultname>.vault.azure.net'
-# KEY_VERSION is required, and can be obtained with the KeyVaultClient.get_key_versions(self, vault_url, key_name) API
-key_bundle = client.get_key(VAULT_URL, KEY_NAME, KEY_VERSION)
-key = key_bundle.key
+key = key_client.get_key("key-name")
+print(key.name)
```

Comment:
wait - isn't this just returing the same name that you just specified to pull from? wasn't this supposed to return the key itself?

Reply:
I do not understand this comment, please clarify.

Reply:
in line 48 you are getting the key by providing the name
in line 49 aren't you just printing pack the same name ? wasn't the goal to retrieve the key value?

Reply:
Perhaps it's sufficient to show creating the `CryptographyClient`, e.g.:
```py
from azure.keyvault.keys.crypto import CryptographyClient

# create a client to perform cryptographic operations with the key
crypto_client = CryptographyClient(key, credential)
```

Reply:
@chlowell @elraikhm What's the decision on this final potential change in the article? Do we really need to add the CryptographyClient now, or can it wait? I'd like to merge this PR asap to close it out, so if I don't hear back on Friday 4/3 I'll go ahead and merge and we can make this edit later on.